src/main/java/it/gov/pagopa/arc/connector/bizevents/BizEventsConnectorImpl.java

```diff
@@ -0,0 +1,38 @@
+package it.gov.pagopa.arc.connector.bizevents;
+
+import feign.FeignException;
+import it.gov.pagopa.arc.connector.bizevents.dto.BizEventsTransactionsListDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+@Service
+@Slf4j
+public class BizEventsConnectorImpl implements BizEventsConnector {
+    private final String apikey;
+    private final  BizEventsRestClient bizEventsRestClient;
+
+    public BizEventsConnectorImpl(@Value("${apiKey.biz-events}") String apikey,
+                                  BizEventsRestClient bizEventsRestClient) {
+        this.apikey = apikey;
+        this.bizEventsRestClient = bizEventsRestClient;
+    }
+
+    @Override
+    public BizEventsTransactionsListDTO transactionsList(String fiscalCode, String continuationToken, int size) {
+        BizEventsTransactionsListDTO bizEventsTransactionsListDTO;
+        try {
+            bizEventsTransactionsListDTO = bizEventsRestClient.transactionsList(apikey, fiscalCode, continuationToken, size);
+        }catch (FeignException e) {
+            bizEventsTransactionsListDTO =
+                    BizEventsTransactionsListDTO
+                            .builder()
+                            .transactions(new ArrayList<>())
+                            .build();
+        }
```

Comment:
we lack logging (and related observability) here.

Reply:
plus:  I don't like very much the "catch all status codes" mode. We could change to "catch the 404 or rethrow the exception", I find it more transparent as if any errors occurs we can see it. 
Later we can think to catch the other status codes when we know what to do